src/app/pages/reportsdashboard/reportsdashboard.component.ts

```diff
@@ -453,6 +484,15 @@ export class ReportsDashboardComponent implements OnInit, OnDestroy, /* HandleCh
       }
     });
 
+    const visibleNoTempDisks = device.filter((dev) => Object.keys(this.disksWithNoTempGraphs).includes(dev));
```

Comment:
This line is triggering a runtime error "Cannot convert undefined or null to object."

Reply:
Should be fixed now.